README.md

```diff
@@ -31,7 +31,7 @@ The default ctrl+b escape is remapped to **ctrl+a**.
 
 **Splits:**
 * **ctrl+a v** split vertically
-* **ctrl+a s** split horizontally
+* **ctrl+a h** split horizontally
```

Comment:
`ctrl+a s` should be correct.  From the configuration file:
```
bind-key s split-window -v
```

The `h` key is for moving between panes (move to the left)

Can you confirm it?

Reply:
On line 13 in tmux.con you can see:

`bind-key v split-window -h`

But inside the current version of the README.md file, I read:

`ctrl+a s split horizontally`

Am I missing something?